srcts/src/components/sidebar.ts

```diff
@@ -242,6 +266,307 @@ class Sidebar {
+
+    const instructions = document.createElement("div");
+    instructions.className = "visually-hidden";
+    instructions.textContent =
+      "Use arrow keys to resize the sidebar, Shift for larger steps, Home/End for min/max width.";
+    handle.appendChild(instructions);
+
+    // Insert handle into the layout container
+    this.layout.container.appendChild(handle);
+    this.layout.resizeHandle = handle;
+
+    this._attachResizeEventListeners();
+  }
+
+  /**
+   * Attach event listeners for resize functionality.
+   * @private
+   */
+  private _attachResizeEventListeners(): void {
+    if (!this.layout.resizeHandle) return;
+
+    const handle = this.layout.resizeHandle;
+
+    // Mouse events
+    handle.addEventListener("mousedown", this._onResizeStart.bind(this));
+    document.addEventListener("mousemove", this._onResizeMove.bind(this));
+    document.addEventListener("mouseup", this._onResizeEnd.bind(this));
+
+    // Touch events for mobile devices
+    handle.addEventListener("touchstart", this._onResizeStart.bind(this), {
+      passive: false,
+    });
+    document.addEventListener("touchmove", this._onResizeMove.bind(this), {
+      passive: false,
+    });
+    document.addEventListener("touchend", this._onResizeEnd.bind(this));
+
+    // Keyboard events for accessibility
+    handle.addEventListener("keydown", this._onResizeKeyDown.bind(this));
+
+    // Prevent text selection during resize
+    handle.addEventListener("selectstart", (e) => e.preventDefault());
+
+    window.addEventListener(
+      "resize",
+      whenChangedCallback(
+        () => this._getWindowSize(),
+        () => this._updateResizeAvailability()
+      )
+    );
+  }
+
+  /**
+   * Check if the sidebar should be resizable in the current state.
+   * @private
+   * @returns {boolean}
+   */
+  private _shouldEnableResize(): boolean {
+    const isDesktop = this._getWindowSize() === "desktop";
+    const notTransitioning = !this.layout.container.classList.contains(
+      Sidebar.classes.TRANSITIONING
+    );
+    const notClosed = !this.isClosed;
+
+    return (
+      // Allow resizing only when the sidebar...
+      isDesktop && notTransitioning && notClosed
+    );
+  }
+
+  /**
+   * Handle resize start (mouse/touch down).
+   * @private
+   * @param {MouseEvent | TouchEvent} event
+   */
+  private _onResizeStart(event: MouseEvent | TouchEvent): void {
+    if (!this._shouldEnableResize()) return;
+
+    event.preventDefault();
+
+    const clientX =
+      "touches" in event ? event.touches[0].clientX : event.clientX;
+
+    this.resizeState.isResizing = true;
+    this.resizeState.startX = clientX;
+    this.resizeState.startWidth = this._getCurrentSidebarWidth();
+
+    // Disable transitions during resize for smooth interaction
+    this.layout.container.style.setProperty("--_transition-duration", "0ms");
+    this.layout.container.classList.add(Sidebar.classes.RESIZING);
+
+    // Change mouse cursor and prevent text selection
```

Comment:
Does this mean that this `handle.addEventListener("selectstart", (e) => e.preventDefault())` listener isn't needed?

Reply:
Yeah, I think so, thanks